src/controllers/breaches.js

```diff
@@ -7,11 +7,13 @@ import { filterBreachDataTypes, appendBreachResolutionChecklist } from '../utils
 import { getSha1 } from '../utils/fxa.js'
 
 async function breachesPage (req, res) {
+  const emailCount = 1 + (req.user.email_addresses?.length || 0)
```

Comment:
So if I'm understanding this correctly, we are adding 1 for the primary email?

Reply:
@mansaj yes, correct.  I will add a comment to make this more clear.